crazyflie_sim/crazyflie_sim/visualization/README.md

```diff
@@ -0,0 +1,88 @@
+# Visualizations
+
+## Blender
+
+### Dependencies
+
+- `bpy`
+- `rowan`
+- `numpy` 
+- `yaml` 
+
+### Output
+- Takes pictures from robots' perspectives
+- Saves pictures in working directory in `simulation_results/<date-and-time>/Raw-Dataset` with the following folder structure
+
+    ```
+    Raw-Dataset
+    ├── cf1
+    │   ├── calibration.yaml
+    │   ├── cf0.csv
+    │   ├── cf0_00000.jpg
+    │   ├── cf0_00001.jpg
+    │   └── ...
+    ├── ...
+    └── cfn
+        ├── calibration.yaml
+        ├── cfn.csv
+        ├── cfn_00000.jpg
+        ├── cfn_00001.jpg
+        └── ...
```

Comment:
For the public version, can we use a simple flat output folder instead? Or at least get rid of the "Raw-Dataset" subfolder.